### PR TITLE
payment checkout ui added

### PR DESCRIPTION
**Pull request by Varsani2520**
@himanshu-03 ,
i added payment page  #660 

image

### CODE REVIEW

**Comment by Varsani2520** [comment]
@himanshu-03 ,check it and merged it also add the level label

File: src/cart.html [hunk 1]
```diff
@@ -313,7 +313,11 @@ <h1>YOUR CART</h1>
         </div>
         </div>
             <div id="subtotal" class="carttext"></div>
-            <div><button class="order_now_button">ORDER NOW</button></div>
+            <div>
+              <a href="/krishiconnect/src/payment.html">
```
Comment (himanshu-03) [inline src/cart.html:317]: provide relative path over here `./payment.html` it should work fine then. Right now its not visible.

Comment (himanshu-03) [reply src/cart.html:317]: Try clicking on the deploy link and see if its working:

Link: https://vercel.live/open-feedback/krishiconnect-git-fork-varsani2520-p-8dae51-anushkabhs-projects.vercel.app?via=pr-comment-visit-preview-link&passThrough=1

Push your changes and then try accessing this link.

**Review by himanshu-03 — CHANGES_REQUESTED** [review]

File: src/cart.html (hunk 1 above)
Comment (Varsani2520) [reply src/cart.html:317]: ok

**Comment by Varsani2520** [comment]
@himanshu-03  ,now check it ,
i added ./payment.html

**Comment by Varsani2520** [comment]
@himanshu-03 ,please merged it

**Comment by Varsani2520** [comment]
@RitiChandak ,please merged it

**Comment by sanjay-kv** [comment]
@himanshu-03 let the contributor know why the delay or any issues. Thanks

**Comment by Varsani2520** [comment]
@himanshu-03 ,please merged it with level label i request you 

@sanjay-kv ,please check this PR ,mentor is not give any response i do everything as they said but now not give any response this is not fair

**Comment by Varsani2520** [comment]
@himanshu-03 ,please merged it

**Comment by RitiChandak** [comment]
Hi @Varsani2520 and @sanjay-kv, we are extremely sorry for the delay.

**Pull request merged**